src/main/java/CenimaWorkers/BaseWorkers.java

```diff
@@ -0,0 +1,38 @@
+    package CenimaWorkers;
+
+    import java.util.Calendar;
+
+    public class BaseWorkers {
```

Comment:
Sformatuj proszę ten kod https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html

CTRL+ALT+L